workflow/scripts/postprocess_go_enrichment.py

```diff
@@ -0,0 +1,47 @@
+import pandas as pd
+
+
+def string_to_tuple(string):
+    """Convert a string representation of a tuple to an actual tuple of integers."""
+    return tuple(map(int, string.replace('(', '').replace(')', '').replace(' ', '').split(',')))
```

Comment:
Have you checked, whether this python functionality can do this for you:

https://stackoverflow.com/a/9763133